requirements.txt

```diff
@@ -1,3 +1,6 @@
+-f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+--pre
+
```

Comment:
do we need this here ? Wont torch-mlir bring in the right version ?

Reply:
Without it something is pulling in Cuda stuff, and I'm not sure what. I can set up a script to knock out lines until I find what's doing it.

Reply:
Many of the packages like brevitas, timm, pytorch-lightning, require a torch. If we don't add this then they won't fetch the CPU torch, resulting in Nvidia packages getting fetched. This won't result in an issue since the torch-mlir gets installed after this, and it would install the appropriate torch package. @powderluv